android/src/main/java/vn/luongvo/kmm/survey/android/ui/common/RatingBar.kt

```diff
@@ -51,24 +56,47 @@ fun HeartRatingBar(
     )
 }
 
+@Composable
+fun SmileyRatingBar(
+    modifier: Modifier = Modifier,
```

Comment:
Please move to modifier to the third parameter

Reply:
Fixed in https://github.com/luongvo/kmm-survey/pull/77/commits/70e3cff423e7fc6056d430b15c75b29d832f5f29